contribution_plan/gql/gql_mutations/payment_plan_mutations.py

```diff
@@ -20,6 +21,8 @@ def _validate_mutation(cls, user, **data):
         if type(user) is AnonymousUser or not user.id or not user.has_perms(
                 ContributionPlanConfig.gql_mutation_create_paymentplan_perms):
             raise ValidationError("mutation.authentication_required")
+        if PaymentPlanService.check_unique_code(data['code']):
+            raise ValidationError("mutation.payment_plan_code_duplicated")
```

Comment:
This will not fetch any translation from .po file. You need to gettext, which is usually imported as _ in other places.